recipes/numpy_quaddtype/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "numpy_quaddtype" %}
+{% set version = "0.0.4" %}
+{% set python_req = "python >=3.10,<3.13" %}
```

Comment:
I think the complaint is coming from the version constraints here

Reply:
Yes, I set that intentionally as NumPy2.0 supports `3.9-3.12`

Reply:
@conda-forge/help-python any chance you can point Swayam at the right way to handle this?

I think all you need to do is depend on NumPy, you don't need to list any constraints that should be handled by that package. I'm not expert though, so let's see if any of the conda-forge maintainers want to help.